changelog.md

```diff
@@ -6,12 +6,24 @@
 		 - Users have the default perms without needing groups.
 	 - Negative perms can now be used to remove permissions from a user.
 	 - Group hierarchy has been implemented meaning groups higher will override groups below them.
+	 - Clone and rename have been added for groups
  - Buttons!
 	 - `_song` and `_skip` have buttons now. More to come in the future
 	 - Pages now have buttons for easy navigation.
  - Pages
 	 - They look nicer.
 	 - Some back-end work.
+ - Nino
+	 - It's an anti invite system for your server
+	 - You can find everything you need for nino in the `_nino` command
+ - Jumbo
+	 - Ever wanted bigger emojies? Now you can with the jumbo command.
+	 - Supports up the 5 emojie at once.
+	 - Gives info on emojies
+	 - Note: Only works on custom emojies
```

Comment:
It only works for emotes currently, emotes are ones uploaded by guilds not the default emojis.